cypress/platform/viewer.js

```diff
@@ -1,4 +1,4 @@
-import { Base64 } from 'js-base64';
+import { Base64 } from '../../node_modules/js-base64';
```

Comment:
This is a super weird issue, maybe it's an issue with how Cypress/Vite are interacting?

Still, if it works, it works :)

In the long-term, maybe it's worth replacing the `js-base64` with the official JavaScript [`btoa()`](https://developer.mozilla.org/en-US/docs/Web/API/btoa)

Reply:
Done.